src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -0,0 +1,210 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Units;
+using System;
+
+namespace Iot.Device.Common
+{
+    /// <summary>
+    /// Helpers for weather
+    /// </summary>
+    public static class WeatherHelper
+    {
+        #region TemperatureAndRelativeHumidity
+        // Formulas taken from http://www.reahvac.com/tools/humidity-formulas/
```

Comment:
per offline conversation we cannot use this as a source because of its non-permissive license. The other link is ok to use. We need to find other source with permissive license in order to use it :disappointed:

Reply:
Done.